cico_setup.sh

```diff
@@ -49,17 +49,17 @@ function cleanup_env {
 }
 
 function deploy() {
-  export REGISTRY="push.registry.devshift.net"
+  export REGISTRY="quay.io"
   export PLUGINS='work-in-progress test-keeper pr-sanitizer'
 
   if [ "${TARGET}" = "rhel" ]; then
     export DEPLOY_DOCKERFILE='Dockerfile.deploy.rhel'
-    export DOCKER_REPO="osio-prod/ike-prow-plugins"
+    export DOCKER_REPO="openshiftio/rhel-ike-prow-plugins"
   fi
 
   # Login first
-  if [ -n "${DEVSHIFT_USERNAME}" -a -n "${DEVSHIFT_PASSWORD}" ]; then
-    docker login -u ${DEVSHIFT_USERNAME} -p ${DEVSHIFT_PASSWORD} ${REGISTRY}
+  if [ -n "${QUAY_USERNAME}" -a -n "${QUAY_PASSWORD}" ]; then
+    docker login -u ${QUAY_USERNAME} -p ${QUAY_PASSWORD} ${REGISTRY}
```

Comment:
I would suggest using `IMG_REPO` prefix instead of `QUAY`. This will then have a proper meaning for any repo we use.